### ISSUE TITLE
Article feedback

### ISSUE THREAD

**Issue by remojansen** [timwhit/enterprise-node-typescript#1]
Hi, 

I'm the author of inversify.

I just found your article http://126kr.com/article/7vyrb344z0l and I want to say thanks for taking the time to write about it. I really appreciate when the community spread the word about it 😄 

Your article was very good but maybe you are not aware of https://github.com/inversify/inversify-express-utils

It would make your express routing much more declarative. You can find a few examples at https://github.com/inversify/inversify-express-example

Keep up the nice work 👍 

Remo

**Comment by timwhit** [comment]
Thanks Remo. I actually saw your library after I wrote the article when I
was helping someone get my sample repo setup for a client project. However,
he ended up using Hapi instead of Express on that project so we couldn't
use the express-utils.

I have no idea what 126kr.com is, but I think they scraped the article from
here:
http://consoledotblog.com/2017/01/typescript-node-js-enterprise-patterns/

It should be published to my companies blog next Monday:
https://www.slalom.com/thinking
